config/locales/future_kids.de.yml

```diff
@@ -314,14 +314,30 @@ de:
   
   self_registrations_mailer:
     user_registered:
-      subject: New %{user_type} registered
-      body: "New %{user_type} registered:"
-
+      subject: Neuer %{user_type} registriert
```

Comment:
@panterch Please check these translations, it's just the Google translate output.